SignNow.Net.Test/AcceptanceTests/GetTokenByAuthCodeAsyncTest.cs

```diff
@@ -0,0 +1,157 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignNow.Net;
+using SignNow.Net.Model;
+using SignNow.Net.Test;
+using SignNow.Net.Test.Context;
+using System;
+using System.Threading.Tasks;
+
+namespace AcceptanceTests
+{
+    [TestClass]
+    public class GetTokenByAuthCodeAsyncTest : ApiTestBase
+    {
+        private const string BAD_REQUEST = "Bad Request";
+
+        private CredentialModel clientInfo, userCredentials;
+        private OAuth2Service authObjectParam2, authObjectParam3;
+        private string authCode = "fac566fb5d926c7dd590f6eea7f8b6c10cbe469b";
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            clientInfo = new CredentialLoader(ApiBaseUrl).GetCredentials();
+            userCredentials = new CredentialLoader(ApplicationBaseUrl).GetCredentials();
+            authObjectParam2 = new OAuth2Service(clientInfo.Login, clientInfo.Password);
+            authObjectParam3 = new OAuth2Service(ApiBaseUrl, clientInfo.Login, clientInfo.Password);
+        }
+
+        [TestMethod]
+        public void Ctor2Params_TokenRetrieving_Success_ScopeAll()
```

Comment:
"Success" tests will never pass with hardcoded auth code. You would better remove them and place a \\TODO: to create automatic auth code retrieval for such a tests